client/src/components/Form/Form.js

```diff
@@ -102,10 +104,12 @@ Form.propTypes = {
     value: PropTypes.any,
     type: PropTypes.string,
   })),
+  formTag: PropTypes.node,
```

Comment:
`node` is pretty broad. Is that deliberate? I guess it means you could inject a component right? If that is the intention then all good

Reply:
Yeah, I'm not 100% if it's the correct thing, it just seemed the closest thing to me. The idea is to be able to specify an HTML tag, _or_ supply a custom component, yes :)

Reply:
I would recommend using `formTag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),` nowadays...
using `PropTypes.node` normally denotes that you'll output it in the render straight away: `<div>{this.props.formStuff}</div>`, which is perfect for `children`

Reply:
Neat thanks, I'll update :)